### ISSUE TITLE
Write test case for illegal sprite priority values

### ISSUE THREAD

**Issue by dirkwhoffmann** [dirkwhoffmann/vAmiga#157]
Desert Strike sets the sprite priority bits in BPLCON2 to 7 which is an invalid value. Find out how this value is interpreted and fix `updateSpritePriorities(uint16_t bplcon2)` accordingly:

```
void
Denise::updateSpritePriorities(uint16_t bplcon2)
{
    switch (bplcon2 & 0b111) {

        case 0: prio1 = Z_0; break;
        case 1: prio1 = Z_1; break;
        case 2: prio1 = Z_2; break;
        case 3: prio1 = Z_3; break;
        case 4: prio1 = Z_4; break;

        default:
            // TODO: Check the effect of illegal values
            // assert(false);
            break;
    }

```

**Comment by mithrendal** [comment]
This is again a showcase why we are so desperately in need of your assertions.  They enable us to take over control when things become uncontrolled and then set back and think about how we like to handle these invalid situations ...  Very nice Dirk !!

**Comment by dirkwhoffmann** [comment]
Corresponding tests: sprites5 to sprites7

**Comment by dirkwhoffmann** [comment]
TODO: Fix #167, #168 first

**Comment by dirkwhoffmann** [comment]
Done
